Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/overlay/InputOverlay.java

```diff
@@ -147,6 +149,9 @@ public boolean onTouch(View v, MotionEvent event)
 				{
 					NativeLibrary.onGamePadEvent(NativeLibrary.TouchScreenDevice, button.getId(), ButtonState.RELEASED);
 				}
+
+				button.setPressedState(false);
+				invalidate();
```

Comment:
if user press down a button and start moving outside the boundaries of that button and release the touch, that button will stuck at pressed state unless the user pressed it again. This will release all pressed button regardless of where the release even happened.